src/libraries/System.Runtime.Experimental/ref/System.Runtime.Experimental.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>System.Runtime</AssemblyName>
```

Comment:
We need to make sure this doesn't leak into the product.  Set these properties (perhaps in the props file) to false to avoid that:
https://github.com/dotnet/runtime/blob/d953229d5429d2ffde833740dd481aab864d3e0c/src/libraries/Directory.Build.targets#L46-L51

Reply:
Fixed.